parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -117,7 +117,9 @@
     },
 {{end}}
   {
-    {{if .IsManagedDisks}}
+    {{if IsOpenShift}}
+      "apiVersion": "[variables('apiVersionVMDefault')]",
```

Comment:
@jackfrancis @CecileRobertMichon 
so, we need a newer apiversion to handle including the "plan" fields.  We're guessing we can't just increase apiVersionDefault.  Are you OK with adding apiVersionVMDefault?  Should the name be OpenShift specific?

Reply:
@jackfrancis @CecileRobertMichon please ignore the above questions ;)